modules/Calendar4You/CalendarUtils.php

```diff
@@ -441,12 +441,14 @@ function transferForAddIntoTitle($type, $row, $CD) {
 		$value = getTranslatedString($Cal_Data[1],'Calendar');
 	else
 		$value = $Cal_Data[1];
-
+	if ($CD["fieldname"] == "location")  {
```

Comment:
Ah this would be a nice chance for me to play with the Global Variable. Will test this and report back.